scss/6-components/_notice-banner.scss

```diff
@@ -0,0 +1,40 @@
+.cads-notice-banner {
+    @include cads-border(
+        l,
+        $cads-border-width-large,
+        0,
+        $cads-language__focus-colour
```

Comment:
should be `$cads-language__brand-secondary` otherwise, perfect 👍

Reply:
Yes it should!